### ISSUE TITLE
Unable to reproduce the same results reported in the paper

### ISSUE THREAD

**Issue by AmeenAli** [microsoft/2D-TAN#13]
Hello <br>
I am trying to reproduce the results mentioned in the paper for the activitynet c3d features.
and I am not getting the same results, the results I am getting are as follows : 
[https://imgur.com/a/AYrJ4hi](url)

I didnt edit the expermints config file at all (by the way is the 2D-TAN-64x64-K9L4-pool.yaml file supposed to give the results mentioned in table 3) ?

Anyidea what might be wrong?
also I didn't find any refrences for the loss function defined in : [https://github.com/microsoft/2D-TAN/blob/master/lib/models/loss.py](url)
so how are you exactly calculating the loss?

**Comment by Sy-Zhang** [comment]
> Hello
> 
> I am trying to reproduce the results mentioned in the paper for the activitynet c3d features.
> and I am not getting the same results, the results I am getting are as follows :
> [https://imgur.com/a/AYrJ4hi](url)
> 
> I didnt edit the expermints config file at all (by the way is the 2D-TAN-64x64-K9L4-pool.yaml file supposed to give the results mentioned in table 3) ?
> 
> Anyidea what might be wrong?
> also I didn't find any refrences for the loss function defined in : [https://github.com/microsoft/2D-TAN/blob/master/lib/models/loss.py](url)
> so how are you exactly calculating the loss?

Due to the randomness during training, you can get similar result with mine but not exactly the same.
From what you show to me, the training set is still underfitting, you need more epoch for training.

The loss function is called [here](https://github.com/microsoft/2D-TAN/blob/9abb4f73aadebdcf6cb4af5349b9e9fd621311cd/moment_localization/train.py#L149)
